src/fuzz_introspector/analysis.py

```diff
@@ -413,14 +413,16 @@ def detect_branch_level_blockers(
 
         # Catch exceptions in case some of the string splitting fails
         try:
-            function_name, rest_string = branch_string.split(':')
+            tmp_idx = branch_string.rfind(':')
```

Comment:
Can we just use rsplit instead, with maxsplit=1 ? 

https://docs.python.org/3/library/stdtypes.html#str.rsplit

Reply:
Done.